src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1827,12 +1827,13 @@ internal enum ErrorCode
 
         ERR_PartialMethodReturnTypeDifference = 8817,
         ERR_PartialMethodRefReturnDifference = 8818,
-        WRN_NullabilityMismatchInReturnTypeOnPartial = 8819,
+        // Available = 8819,
```

Comment:
Is this diagnostic available? Haven't we shipped stable versions using this diagnostic ID, and so want to avoid reusing the ID for a different diagnostic? (I think in the past we've denoted this by simply commenting out the line.)

Reply:
We could mark this error code as unavailable, but the error code was only added in 16.7 for a preview feature.

---
In reply to: [486000323](https://github.com/dotnet/roslyn/pull/47576#discussion_r486000323) [](ancestors = 486000323)